src/manager.rs

```diff
@@ -30,18 +24,14 @@ use std::sync::{
     RwLock,
 };
 
-use error::{
-    StoreError,
-};
+use error::StoreError;
 
-use ::Rkv;
+use Rkv;
 
 /// A process is only permitted to have one open handle to each Rkv environment.
 /// This manager exists to enforce that constraint: don't open environments directly.
 lazy_static! {
-    static ref MANAGER: RwLock<Manager> = {
-        RwLock::new(Manager::new())
-    };
+    static ref MANAGER: RwLock<Manager> = { RwLock::new(Manager::new()) };
```

Comment:
This is fair enough, although it would be even simpler and more readable to also remove the brackets:

```rust
    static ref MANAGER: RwLock<Manager> = RwLock::new(Manager::new());
```

Unsure if that's configurable, but it isn't a big deal. I'm ok either way.